core/src/main/kotlin/ch/ergon/dope/resolvable/expression/unaliased/type/numberfunction/RoundExpression.kt

```diff
@@ -0,0 +1,20 @@
+package ch.ergon.dope.resolvable.expression.unaliased.type.numberfunction
+
+import ch.ergon.dope.resolvable.expression.TypeExpression
+import ch.ergon.dope.resolvable.expression.unaliased.type.toDopeType
+import ch.ergon.dope.validtype.NumberType
+
+class RoundExpression(value: TypeExpression<NumberType>, digits: TypeExpression<NumberType>? = null) :
+    NumberFunctionExpression("ROUND", value, digits)
+
+fun round(value: TypeExpression<NumberType>) = RoundExpression(value)
+
+fun round(value: Number) = round(value.toDopeType())
+
+fun round(value: TypeExpression<NumberType>, digits: TypeExpression<NumberType>) = RoundExpression(value, digits)
+
+fun round(value: TypeExpression<NumberType>, digits: Number) = round(value, digits.toDopeType())
+
+fun round(value: Number, digits: TypeExpression<NumberType>) = round(value.toDopeType(), digits)
+
+fun round(value: Number, digits: Number) = round(value.toDopeType(), digits.toDopeType())
```

Comment:
Here you have spaces between the functions. I would at least unify both styles into one and stick to that.

Reply:
Done, unified to blank lines